lib/oc-tests/test_shareFile.py

```diff
@@ -176,7 +176,7 @@ def shareeOne(step):
     list_files(d)
 
     shared = reflection.getSharedObject()
-    if sharePermissions == (OCS_PERMISSION_READ | OCS_PERMISSION_SHARE):
+    if not sharePermissions & OCS_PERMISSION_UPDATE:
       expect_not_modified(os.path.join(d,'TEST_FILE_MODIFIED_USER_SHARE.dat'), shared['md5_sharer'])
```

Comment:
Hmm, I think for read-only files the client will keep the local version. Then it will try and continue syncing it over and over again, in case the permissions changed. It might blacklist it after a while.

So it is correct to have a check that makes sure that the remote file did NOT change, neither for the owner user1 not for the recipient user2. (but the local file stays changed)